server/historian/packages/historian-base/src/runnerFactory.ts

```diff
@@ -48,9 +49,21 @@ export class HistorianResourcesFactory implements utils.IResourcesFactory<Histor
         const riddlerEndpoint = config.get("riddler");
         const riddler = new historianServices.RiddlerService(riddlerEndpoint, tenantCache);
 
+        const throttleMaxRequestsPerMs = config.get("throttling:maxRequestsPerMs") as number || undefined;
```

Comment:
add default values instead of undefined.
for reference, take a look at Zach's Alfred pr. :)

Reply:
Actually, my PR introduced changes to `Throttler` and `ThrottlerHelper` instead, which made these arguments optional with defaults 😃

Reply:
Yes, I noticed that Zach has updated default values in the throttler library, and here undefined is used, if it ends up with undefined https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts#L171